migrations/3_batch_auction.js

```diff
@@ -1,16 +1,13 @@
 /*eslint no-undef: "off"*/
 
 const BiMap = artifacts.require("IdToAddressBiMap.sol")
-const IterableAppendOnlySet = artifacts.require("IterableAppendOnlySet.sol")
 const SnappBaseCore = artifacts.require("SnappBaseCore.sol")
 const SnappAuction = artifacts.require("SnappAuction.sol")
 
 module.exports = async function (deployer) {
   await deployer.deploy(BiMap)
-  await deployer.deploy(IterableAppendOnlySet)
 
   await deployer.link(BiMap, SnappBaseCore)
-  await deployer.link(IterableAppendOnlySet, SnappBaseCore)
```

Comment:
So does this somehow imply that it does not need to be linked?

Reply:
Not with SnappBaseCore. IterableAppendOnlySet is only part of the StablecoinConverter contract and as such needs to be linked against that.